qt/scientific_interfaces/EnggDiffraction/EnggDiffGSASFittingModel.cpp

```diff
@@ -8,6 +8,8 @@
 
 using namespace Mantid;
 
+Q_DECLARE_METATYPE(Mantid::API::IAlgorithm_sptr)
```

Comment:
This is already defined in `MantidAlgorithmMetatype.h`, perhaps include that instead?

Reply:
Same in other places...